framework/src/modules/token/commands/transfer.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { BaseCommand } from '../../base_command';
+import { CommandExecuteContext } from '../../../node/state_machine';
+import { TokenAPI } from '../api';
+
+interface Params {
+	amount: bigint;
+	recipientAddress: Buffer;
+	data: string;
+}
+
+export class TransferCommand extends BaseCommand {
+	public name = 'transfer';
+	public id = 0;
+	public schema = {
```

Comment:
I think `schema` here can be moved as `transferCommandParamsSchema` to `schemas.ts` similar to other module commands